src/.prettierrc

```diff
@@ -1,8 +1,9 @@
-# .prettierrc
```

Comment:
Why this file changed?

Reply:
I wrote , prettier didn't work and I fixed that